ui/app/AppLayouts/Profile/popups/ExemptionNotificationsModal.qml

```diff
@@ -0,0 +1,159 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
+
+import utils 1.0
+import shared.panels 1.0
+
+import "../controls"
+
+StatusModal {
+    id: root
+
+    property var notificationsStore
+    property var item: ({
+        name: "",
+        image: "",
+        color: "",
+        customized: false,
+        type: Constants.settingsSection.exemptions.community,
+        muteAllMessages: false,
+        personalMentions: Constants.settingsSection.notifications.sendAlertsValue,
+        globalMentions: Constants.settingsSection.notifications.sendAlertsValue,
+        otherMessages: Constants.settingsSection.notifications.turnOffValue
+    })
+
+    header.title: qsTr("%1 exemption").arg(root.item.name)
+    header.image.source: root.item.image
+    header.icon: StatusIconSettings {
+        // Once we introduce StatusSmartIdenticon in popup header, we should use the folowing
+//        color: root.item.type === Constants.settingsSection.exemptions.oneToOneChat?
+//                   Theme.palette.userCustomizationColors[Utils.colorIdForPubkey(root.item.itemId)] :
+//                   root.item.color
+        // until then the following is used
+        background.color: root.item.type === Constants.settingsSection.exemptions.oneToOneChat?
+                              Theme.palette.userCustomizationColors[Utils.colorIdForPubkey(root.item.itemId)] :
```

Comment:
I wonder if the correct color shouldn't be populated by the backend?

Reply:
@jrainville as I see so far on other places that's defined on the qml side (part of `Theme`), determined by `Theme.palette.userCustomizationColors`.